project/Dependencies.scala

```diff
@@ -12,51 +12,33 @@ import sbt._
 object Dependencies {
 
   object Versions {
-    val crossScala   = Seq("2.13.3", "2.12.12")
+    val crossScala = Seq("2.13.3", "2.12.12")
     val scalaVersion = crossScala.head
+    val testCommon = "0.2.3"
```

Comment:
Non blocking: the variable is used only once as far as I can see so it doesn't add any value

Reply:
This is the standard way we are moving to of writing dependencies